src/Integration/MefServices/MefSonarQubeService.cs

```diff
@@ -36,6 +40,16 @@ public class MefSonarQubeService : SonarQubeService
         public MefSonarQubeService()
             : base(new SonarQubeClient(new HttpClientHandler(), TimeSpan.FromSeconds(100)))
         {
+            // TODO: Remove the following code which only purpose is to enable a quick testing while deciding of the direction
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this 'TODO' comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1135)